test/export-api-def.test.js

```diff
@@ -0,0 +1,116 @@
+/*global describe, beforeEach, it */
+'use strict';
+var path = require('path');
+var helpers = require('yeoman-generator').test;
+var fs = require('fs');
+var expect = require('chai').expect;
+var wsModels = require('loopback-workspace').models;
+var common = require('./common');
+var yaml = require('js-yaml');
+var install = require('strong-cached-install');
+var SANDBOX =  path.resolve(__dirname, 'sandbox');
+var PKG_CACHE = path.resolve(__dirname, '..', '.pkgcache');
+
+describe('loopback:export-api-def generator', function() {
+  this.timeout(30 * 60 * 1000); // 30 minutes
+  before(common.resetWorkspace);
+
+  before(function createSandbox(done) {
+    helpers.testDirectory(SANDBOX, done);
+  });
+
+  before(function createProject(done) {
+    common.createDummyProject(SANDBOX, 'test-app', done);
+  });
+
+  before(function createProductModel(done) {
+    var test = this;
+    wsModels.ModelDefinition.create(
+      {
+        name: 'Product',
+        facetName: 'common'
+      },
+      function(err, model) {
+        test.Model = model;
+        done(err);
+      });
+  });
+
+  before(function installDependencies(done) {
+    install(SANDBOX, PKG_CACHE, ['dependencies'], done);
+  });
+
+  it('JSON format', function(done) {
```

Comment:
`it('produces JSON format')` would be a better test name. Same for YAML test below.